autopilot/autopilot.go

```diff
@@ -701,7 +730,7 @@ func (ap *Autopilot) stateHandlerGET(jc jape.Context) {
 		return
 	}
 
-	jc.Encode(api.AutopilotStateResponse{
+	ap.writeResponse(jc, http.StatusOK, AutopilotStateResp(api.AutopilotStateResponse{
```

Comment:
This won't work with our static code analysis tool `jape`.  It expects `jc.Encode` to be called in the handler right now. So without updating that this won't make it through the CI.

Reply:
We should seriously consider extending `jape` to support this or maybe try and write some kind of middleware for it. It won't work otherwise. Maybe we can make `jape.Context` an interface and then we can swap it (using a middleware) with one that's capable of handling `prometheus` response types. E.g. test the interface in `Encode` on some kind of interface that implements `toPrometheus` and handle it accordingly